docs/zh/latest/plugins/google-cloud-logging.md

```diff
@@ -137,6 +139,7 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 \
         "google-cloud-logging": {
             "auth_config":{
                 "project_id":"apisix",
+                "client_email":"your service account email@apisix.iam.gserviceaccount.com",
```

Comment:
The `token_uri` is missing here, could you help to add?

Reply:
There’s a default value, so  `token_uri` is not a required field in minimum settings